db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170319214240) do
+ActiveRecord::Schema.define(version: 20170613071826) do
```

Comment:
any specific way to do that?
what I tried so far is to create a single migration file and I combined **up**, **down** and **change** from the separate files together and I also deleted the old migration files.. but it doesn't seem it is working!

here's the final file (20170616175054_acts_as_taggable_on_migration.rb):

```
# This migration comes from acts_as_taggable_on_engine (Note: they were manually combined in a single file)
class ActsAsTaggableOnMigration < ActiveRecord::Migration[5.0]
    def self.up
        create_table :tags do |t|
            t.string :name
            t.boolean :is_approved, default: false
        end

        create_table :taggings do |t|
            t.references :tag

            # You should make sure that the column created is
            # long enough to store the required class names.
            t.references :taggable, polymorphic: true
            t.references :tagger, polymorphic: true

            # Limit is created to prevent MySQL error on index
            # length for MyISAM table type: http://bit.ly/vgW2Ql
            t.string :context, limit: 128

            t.datetime :created_at
        end

        add_index :taggings, :tag_id
        add_index :taggings, [:taggable_id, :taggable_type, :context]

        add_index :tags, :name, unique: true
        remove_index :taggings, :tag_id if index_exists?(:taggings, :tag_id)
        remove_index :taggings, [:taggable_id, :taggable_type, :context]
        add_index :taggings,
                [:tag_id, :taggable_id, :taggable_type, :context, :tagger_id, :tagger_type],
                unique: true, name: 'taggings_idx'
        add_column :tags, :taggings_count, :integer, default: 0

        ActsAsTaggableOn::Tag.reset_column_information

        ActsAsTaggableOn::Tag.find_each do |tag|
        ActsAsTaggableOn::Tag.reset_counters(tag.id, :taggings)
        end

        add_index :taggings, [:taggable_id, :taggable_type, :context]
        end # up

    def self.down
        drop_table :taggings
        drop_table :tags
        remove_index :tags, :name

        remove_index :taggings, name: 'taggings_idx'

        add_index :taggings, :tag_id unless index_exists?(:taggings, :tag_id)
        add_index :taggings, [:taggable_id, :taggable_type, :context]
        remove_column :tags, :taggings_count
        remove_index :taggings, [:taggable_id, :taggable_type, :context]
    end # down


    def change
        add_index :taggings, :tag_id
        add_index :taggings, :taggable_id
        add_index :taggings, :taggable_type
        add_index :taggings, :tagger_id
        add_index :taggings, :context

        add_index :taggings, [:tagger_id, :tagger_type]
        add_index :taggings, [:taggable_id, :taggable_type, :tagger_id, :context], name: 'taggings_idy'
    end # change
end # class
```